sdk/tables/azure-data-tables/pom.xml

```diff
@@ -38,6 +38,11 @@ Licensed under the MIT License.
       <artifactId>azure-core</artifactId>
       <version>1.17.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
```

Comment:
Is this dependency just for the sample that includes `DefaultAzureCredential`?

Reply:
Yes, but it seems I need to scope it to be included only in tests, as @alzimmermsft pointed out below.